service/static/index.html

```diff
@@ -114,10 +114,12 @@ <h4>Address Information:</h4>
             <div class="form-horizontal">
               <div class="form-group">
                 <div class="col-sm-offset-2 col-sm-10">
-                  <button type="submit" class="btn btn-primary" id="addr-search-btn">Search for Customer Addresses</button>
+                  <button type="submit" class="btn btn-primary" id="addr-query-btn">List A Customer' Addresses</button>
```

Comment:
For customers, it's setup so that list is the same as searching without any of the query parameter fields filled in. wondering if we should do the same here

Reply:
> For customers, it's setup so that list is the same as searching without any of the query parameter fields filled in. wondering if we should do the same here

Yes I just found out that I misunderstood the function I needed to implement. I will fix it.